(base) ]0;~/nlp-writer-scaffold-main/app[01;34m~/nlp-writer-scaffold-main/app[00m$ clearpython main.py[K[K[K[K[K[K[Ksimple_site.py
Try to open

    https://cocalc1.ai-camp.org/00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12345/


 * Serving Flask app "simple_site" (lazy loading)
 * Environment: production
[31m   WARNING: This is a development server. Do not use it in a production deployment.[0m
[2m   Use a production WSGI server instead.[0m
 * Debug mode: on
Traceback (most recent call last):
  File "simple_site.py", line 35, in <module>
    app.run(host = '0.0.0.0', port = port, debug=True)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 990, in run
    run_simple(host, port, self, **options)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/werkzeug/serving.py", line 1030, in run_simple
    s.bind(server_address)
OSError: [Errno 98] Address already in use
(base) ]0;~/nlp-writer-scaffold-main/app[01;34m~/nlp-writer-scaffold-main/app[00m$ clear
[H[J(base) ]0;~/nlp-writer-scaffold-main/app[01;34m~/nlp-writer-scaffold-main/app[00m$ clearpython simple_site.py
Try to open

    https://cocalc1.ai-camp.org/00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/


 * Serving Flask app "simple_site" (lazy loading)
 * Environment: production
[31m   WARNING: This is a development server. Do not use it in a production deployment.[0m
[2m   Use a production WSGI server instead.[0m
 * Debug mode: on
 * Running on http://0.0.0.0:12346/ (Press CTRL+C to quit)
 * Restarting with inotify reloader
Try to open

    https://cocalc1.ai-camp.org/00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/


 * Debugger is active!
 * Debugger PIN: 293-173-532
127.0.0.1 - - [20/Jul/2021 18:34:49] "[35m[1mGET /00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/nlp-writer-scaffold-main/app/simple_site.py", line 30, in home
    return render_template('Overview.html', name=name)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/templating.py", line 138, in render_template
    ctx.app.jinja_env.get_or_select_template(template_name_or_list),
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/jinja2/environment.py", line 1068, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/jinja2/environment.py", line 997, in get_template
    return self._load_template(name, globals)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/jinja2/environment.py", line 958, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/jinja2/loaders.py", line 137, in load
    code = environment.compile(source, name, filename)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/jinja2/environment.py", line 757, in compile
    self.handle_exception(source=source_hint)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/jinja2/environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/nlp-writer-scaffold-main/app/templates/Overview.html", line 50, in template
    <ul class="u-nav u-unstyled u-nav-1"><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-2-base" href="{{ url_for('home'),_anchor='sec-1fe0' }}" data-page-id="513338" style="padding: 10px 20px;">Overview</a>
jinja2.exceptions.TemplateSyntaxError: expected token 'end of print statement', got '='
127.0.0.1 - - [20/Jul/2021 18:34:49] "[37mGET /00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
127.0.0.1 - - [20/Jul/2021 18:34:49] "[37mGET /00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
127.0.0.1 - - [20/Jul/2021 18:34:49] "[37mGET /00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
127.0.0.1 - - [20/Jul/2021 18:34:50] "[37mGET /00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
127.0.0.1 - - [20/Jul/2021 18:34:50] "[37mGET /00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
127.0.0.1 - - [20/Jul/2021 18:34:50] "[37mGET /00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
^C(base) ]0;~/nlp-writer-scaffold-main/app[01;34m~/nlp-writer-scaffold-main/app[00m$ python simple_site.py
Try to open

    https://cocalc1.ai-camp.org/00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/


 * Serving Flask app "simple_site" (lazy loading)
 * Environment: production
[31m   WARNING: This is a development server. Do not use it in a production deployment.[0m
[2m   Use a production WSGI server instead.[0m
 * Debug mode: on
 * Running on http://0.0.0.0:12346/ (Press CTRL+C to quit)
 * Restarting with inotify reloader
Try to open

    https://cocalc1.ai-camp.org/00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/


 * Debugger is active!
 * Debugger PIN: 293-173-532
127.0.0.1 - - [20/Jul/2021 18:38:43] "[35m[1mGET /00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/nlp-writer-scaffold-main/app/simple_site.py", line 30, in home
    return render_template('Overview.html', name=name)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/templating.py", line 137, in render_template
    return _render(
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/jinja2/environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/nlp-writer-scaffold-main/app/templates/Overview.html", line 53, in top-level template code
    </li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-2-base" href="{{ url_for('home') }}" data-page-id="513338" style="padding: 10px 20px;">Try it out!</a><div class="u-nav-popup"><ul class="u-h-spacing-20 u-nav u-unstyled u-v-spacing-10 u-nav-2"><li class="u-nav-item"><a class="u-button-style u-nav-link u-white" href="{{ url_for('Tupac_function') }}">Tupac</a>
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/helpers.py", line 370, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 2216, in handle_url_build_error
    reraise(exc_type, exc_value, tb)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/helpers.py", line 357, in url_for
    rv = url_adapter.build(
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/werkzeug/routing.py", line 2179, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.BuildError: Could not build url for endpoint 'Tupac_function'. Did you mean 'static' instead?
127.0.0.1 - - [20/Jul/2021 18:38:43] "[37mGET /00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
127.0.0.1 - - [20/Jul/2021 18:38:43] "[37mGET /00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
127.0.0.1 - - [20/Jul/2021 18:38:43] "[37mGET /00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
127.0.0.1 - - [20/Jul/2021 18:38:43] "[37mGET /00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
127.0.0.1 - - [20/Jul/2021 18:38:44] "[37mGET /00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
^C(base) ]0;~/nlp-writer-scaffold-main/app[01;34m~/nlp-writer-scaffold-main/app[00m$ clear
[H[J(base) ]0;~/nlp-writer-scaffold-main/app[01;34m~/nlp-writer-scaffold-main/app[00m$ clearpython simple_site.pyclear[Kpython simple_site.py
Try to open

    https://cocalc1.ai-camp.org/00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/


 * Serving Flask app "simple_site" (lazy loading)
 * Environment: production
[31m   WARNING: This is a development server. Do not use it in a production deployment.[0m
[2m   Use a production WSGI server instead.[0m
 * Debug mode: on
 * Running on http://0.0.0.0:12346/ (Press CTRL+C to quit)
 * Restarting with inotify reloader
Try to open

    https://cocalc1.ai-camp.org/00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/


 * Debugger is active!
 * Debugger PIN: 293-173-532
127.0.0.1 - - [20/Jul/2021 18:39:16] "[35m[1mGET /00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/ HTTP/1.1[0m" 500 -
Traceback (most recent call last):
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 2464, in __call__
    return self.wsgi_app(environ, start_response)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 2450, in wsgi_app
    response = self.handle_exception(e)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 1867, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/nlp-writer-scaffold-main/app/simple_site.py", line 30, in home
    return render_template('Overview.html', name=name)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/templating.py", line 137, in render_template
    return _render(
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/jinja2/environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/nlp-writer-scaffold-main/app/templates/Overview.html", line 53, in top-level template code
    </li><li class="u-nav-item"><a class="u-button-style u-nav-link u-text-active-palette-1-base u-text-hover-palette-2-base" href="{{ url_for('home') }}" data-page-id="513338" style="padding: 10px 20px;">Try it out!</a><div class="u-nav-popup"><ul class="u-h-spacing-20 u-nav u-unstyled u-v-spacing-10 u-nav-2"><li class="u-nav-item"><a class="u-button-style u-nav-link u-white" href="{{ url_for('Tupac_function') }}">Tupac</a>
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/helpers.py", line 370, in url_for
    return appctx.app.handle_url_build_error(error, endpoint, values)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/app.py", line 2216, in handle_url_build_error
    reraise(exc_type, exc_value, tb)
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/flask/helpers.py", line 357, in url_for
    rv = url_adapter.build(
  File "/projects/00817c4c-2234-4da4-8978-4e3c95b77bdd/miniconda3/lib/python3.8/site-packages/werkzeug/routing.py", line 2179, in build
    raise BuildError(endpoint, values, method, self)
werkzeug.routing.BuildError: Could not build url for endpoint 'Tupac_function'. Did you mean 'static' instead?
127.0.0.1 - - [20/Jul/2021 18:39:17] "[37mGET /00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 200 -
127.0.0.1 - - [20/Jul/2021 18:39:17] "[37mGET /00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 200 -
127.0.0.1 - - [20/Jul/2021 18:39:17] "[37mGET /00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/?__debugger__=yes&cmd=resource&f=jquery.js HTTP/1.1[0m" 200 -
127.0.0.1 - - [20/Jul/2021 18:39:18] "[37mGET /00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/?__debugger__=yes&cmd=resource&f=ubuntu.ttf HTTP/1.1[0m" 200 -
127.0.0.1 - - [20/Jul/2021 18:39:18] "[37mGET /00817c4c-2234-4da4-8978-4e3c95b77bdd/port/12346/?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 200 -
